Resources/public/js/views/ez-dashboardblocksview.js

```diff
@@ -145,7 +145,14 @@ YUI.add('ez-dashboardblocksview', function (Y) {
              * @readOnly
              */
             blocks: {
-                value: [],
+                valueFn: function () {
+                    return [
+                        new Y.eZ.DashboardBlockAllContentView({
+                            priority: 1,
+                            bubbleTargets: this
```

Comment:
You only took half of the https://github.com/ezsystems/PlatformUIBundle/pull/589/files#r63186170 into account so now that the root Location is loaded in the view service:

1. the dashboard blocks view should receive it (ie you have to implement a `_getViewParameters` in the view service and add the corresponding attribute in the dashboard blocks view)
1. then here, the `eZ.DashboardBlockAllContentView` instance should receive that Location, that way the view has everything to directly fire the `contentSearch`  event.
1. with that in place, the plugin becomes useless and can be removed.

(nitpick: always add `,` at the end of those kind of plain object, that makes future diff cleaner and sometimes avoid stupid conflict)